doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1301,7 +1301,7 @@ Notations
 Abbreviations
 ~~~~~~~~~~~~~
 
-.. cmd:: Ltac2 Notation {| @string | @ident } := @ltac2_expr
+.. cmd:: Ltac2 Notation @ident := @ltac2_expr
    :name: Ltac2 Notation (abbreviation)
 
    Introduces a special kind of notation, called an abbreviation,
```

Comment:
At the end of this paragraph, would you add ```(See :tacn:`Ltac2 Notation` for the general description of notations.)```?

Also in the description for ```.. cmd:: Ltac2 Notation {+ @ltac2_scope } {? : @natural } := @ltac2_expr```, would you add ```(Also see :tacn:`Ltac2 Notation (abbreviation)`)``` after the first paragraph?

Thanks.